### PR TITLE
[10.0][MIG] stock_inventory_line_price: to v10

### PR DESCRIPTION
**Pull request by lasley**
* Rename manifest
* Bump versions
* Openerp references to Odoo

v9 migration is in  #275

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by congdpt — APPROVED** [review]

File: stock_inventory_line_price/README.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================
+Standard price at inventory level
+=================================
+
+* This module allows you to handle the standard price at inventory level.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (tafaRU) [inline stock_inventory_line_price/README.rst:11]: this part should be moved under https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#usage

**Review by tafaRU — APPROVED** [review]
Only some nitpicking in the README.rst otherwise :+1:

File: stock_inventory_line_price/README.rst (hunk 1 above)
Comment (lasley) [reply stock_inventory_line_price/README.rst:11]: Hmmm but there isn't a usage section. Does it make sense to add one just for the Runbot link?

**Comment by rousseldenis** [comment]
@lasley @tafaRU @lreficent Can we move forward on this one ?

File: stock_inventory_line_price/views/inventory_view.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        <field name="name">view.inventory.form.inherit</field>
+        <field name="model">stock.inventory</field>
+        <field name="inherit_id" ref="stock.view_inventory_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='line_ids']//field[@name='product_id']" position="after">
+                <field name="theoretical_std_price"/>
+                <field name="standard_price"/>
+            </xpath>
+            <xpath expr="//tree" position="attributes">
+                <attribute name="colors">blue: product_qty != theoretical_qty or theoretical_std_price != standard_price; red: theoretical_qty &lt; 0</attribute>
```
Comment (LoisRForgeFlow) [inline stock_inventory_line_price/views/inventory_view.xml:14]: We are still having the same issue here as in [v9](https://github.com/OCA/stock-logistics-warehouse/pull/275#pullrequestreview-37237048)

File: stock_inventory_line_price/models/stock_inventory_line.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            'price_unit': self.standard_price,
+        })
+        return vals
+
+    @api.multi
+    def _generate_moves(self):
+
+        moves = self.env['stock.move']
+
+        for inventory_line in self:
```
Comment (rousseldenis) [inline stock_inventory_line_price/models/stock_inventory_line.py:37]: @lasley You make this method one as the original one is multi.

You'd better write:

```python
moves = self.env['stock.move'].browse() # Real void recordset
equal_lines = self.filtered(lambda l: l.theoretical_std_price==l.standard_price)
non_equal_lines = self - equal_lines
moves = super(StockInventoryLine, equal_lines)._generate_moves()
for line in non_equal_lines:
    line.product_id.standard_price = line.standard_price
moves += super(StockInventoryLine, non_equal_lines)._generate_moves()
return moves
```

Maximum two calls to super comparing to n calls

File: stock_inventory_line_price/models/stock_inventory.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class StockInventory(models.Model):
+    _inherit = 'stock.inventory'
+
+    @api.multi
+    def _get_inventory_lines_values(self):
+        vals = super(StockInventory, self)._get_inventory_lines_values()
+        for line in vals:
+            product = self.env['product.product'].browse(line['product_id'])
```
Comment (rousseldenis) [inline stock_inventory_line_price/models/stock_inventory.py:15]: Maybe consider browsing all products at one time to avoid performances problems.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review.

Is someone maintaining this as @lasley won't maybe fix it?

**Comment by rousseldenis** [comment]
I suppose we can close this soon

**Comment by LoisRForgeFlow** [comment]
@rousseldenis I think we can do it now, this is old and we all know Lasley is not working in Odoo topics anymore.

**Comment by rousseldenis** [comment]
> @rousseldenis I think we can do it now, this is old and we all know Lasley is not working in Odoo topics anymore.

:+1: I was waiting because I thought @pedrobaeza has access to his former repos.

**Comment by pedrobaeza** [comment]
In fact all repository maintainers have access (including you, Denis). You just need to push to that remote.

**Pull request closed without merging**